samcli/commands/test_runner/init/cli.py

```diff
@@ -0,0 +1,204 @@
+"""
+CLI command for "test-runner init" command
+"""
+from typing import Any, Optional
+import logging
+
+import click
+
+from samcli.cli.main import pass_context
+from samcli.commands.exceptions import NoResourcesMatchGivenTagException
+from samcli.lib.test_runner.generate_env_vars import FargateRunnerArnMapGenerator
+from samcli.lib.test_runner.test_runner_template_generator import FargateRunnerCFNTemplateGenerator
+from samcli.lib.test_runner.invoke_testsuite import FargateTestsuiteRunner
+from samcli.lib.test_runner.utils import query_tagging_api, write_file
+from samcli.lib.utils.boto_utils import get_boto_client_provider_with_config
+from samcli.lib.utils.colors import Colored
+
+SHORT_HELP = "Generates a Test Runner CloudFormation Template in YAML format, as well as a resource-ARN map."
+HELP_TEXT = """
+This command generates a CloudFormation Template file that can be deployed to instantiate a Test Runner Stack.
+
+It also generates a resource-ARN map, where the keys are your resource names in the format of an environment variable name, and the values
+are the corresponding resource ARN. 
+
+For example, 
+
+_MY_LAMBDA_FUNCTION : arn:aws:lambda:us-east-1:123456789012:function:my-lambda-function
+
+This resource-ARN map can be passed to the `sam test-runner run` command which will expose these mappings as environment variables to the Test Runner Fargate instance,
+and thus can be used in test code.
+"""
+
+DEFAULT_RESOURCE_ARN_MAP_FILE_NAME = "test_runner_environment_variables.yaml"
+DEFAULT_TEST_RUNNER_CFN_TEMPLATE_FILE_NAME = "test_runner_template.yaml"
+SUPPORTED_RUNTIMES = ["python3.8"]
+
+LOG = logging.getLogger(__name__)
+COLOR = Colored()
+
+
+@click.command("init", help=HELP_TEXT, short_help=SHORT_HELP)
+@click.option(
+    "--tag-key",
```

Comment:
Are you only allowing one tag to be provided? Is there never a case to provide multiple? 

How would a customer provide multiple keys if they want?

Seems like https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/resourcegroupstaggingapi.html#ResourceGroupsTaggingAPI.Client.get_resources supports multiple keys, so wouldn't this implementation limit our ability to do that since key and values are two different options provided to the command?

Reply:
Right, this should have been noted in the PR. Having the tags specified as two separate options is a temporary solution, and a point of improvement. Ideally, we'd want to mimic the way AWS CLI enters tags, this "shorthand" form, e.g. `Key=string,Values=string,string`. However, [their parser](https://github.com/aws/aws-cli/blob/2f09fcc0e28784affb472d9aa0d3dd2c3ab513de/awscli/shorthand.py) is not something I can easily copy. 

I don't want to implement my own simple version of it for fear of edge cases that produce behavior inconsistent with AWS CLI, as that would be really confusing for customers.

I couldn't find one grepping around, but does SAM CLI have similar shorthand parser?

Reply:
I would check what we do for deploy. This can be something to follow and would make the cli consistent on how tags are done

Reply:
Ah, I found a `CfnTags` class which is awesome. It's not the same format as AWS CLI, but that's fine, as long as this tag option is consistent with other tag options in SAM CLI